__tests__/pet.test.js

```diff
@@ -0,0 +1,218 @@
+const Pet = require('../src/pet');
+
+describe('constructor', () => {
+  let pet;
+  beforeEach(() => {
+   pet = new Pet('Jessie');
+  });
+    it('returns an object', () => {
+      expect(new Pet('Jessie')).toBeInstanceOf(Object);
+    });
+    it('sets the name property', () => {
+      expect(pet.name).toEqual('Jessie');
+    });
+    it('has an initial age of 0', () => {
+      expect(pet.age).toEqual(0);
+    });
+    it('has an initial hunger of 0', () => {
+     expect(pet.hunger).toEqual(0);
+    });
+    it('has initial fitness of 10', () => {
+      expect(pet.fitness).toEqual(10);
+    } )
+  });
+
+  describe('growUp', () => {
```

Comment:
There are still some 'magic' numbers in here in the names of the tests, instead of 'increments the age by 1' could it be 'increments the age by the correct amount'?